<ui version="4.0" >
 <class>RibbonSettingsWidget</class>
 <widget class="QWidget" name="RibbonSettingsWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>233</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Rendering:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QComboBox" name="renderType" >
     <item>
      <property name="text" >
       <string>Backbone</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Lines</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="bondRadiusLabel_2" >
     <property name="text" >
      <string>Radius:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QSlider" name="radiusSlider" >
     <property name="minimum" >
      <number>0</number>
     </property>
     <property name="maximum" >
      <number>20</number>
     </property>
     <property name="singleStep" >
      <number>1</number>
     </property>
     <property name="pageStep" >
      <number>2</number>
     </property>
     <property name="value" >
      <number>10</number>
     </property>
     <property name="sliderPosition" >
      <number>10</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance" >
      <bool>false</bool>
     </property>
     <property name="tickPosition" >
      <enum>QSlider::TicksBothSides</enum>
     </property>
     <property name="tickInterval" >
      <number>4</number>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="QCheckBox" name="useNitrogens" >
     <property name="text" >
      <string>Include Nitrogens</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
